tsdb/batcher.go

```diff
@@ -22,13 +22,16 @@ type PointBatcher struct {
 	wg *sync.WaitGroup
 }
 
-// NewPointBatcher returns a new PointBatcher.
-func NewPointBatcher(sz int, d time.Duration) *PointBatcher {
+// NewPointBatcher returns a new PointBatcher. sz is the batching size,
+// bf is the maximum number of batches that may be pending. d is the time
```

Comment:
Ha, I used to call it `bf` in the code, but forgot to update the comment -- thanks.